### ISSUE TITLE
Exception `TypeError: \'NoneType\' object is not iterable` thrown when no providers have been added 

### ISSUE THREAD

**Issue by kmarcisz** [morpheus65535/bazarr#1891]
**Describe the bug**
With an installation of bazarr that has no providers added when user tries to search for subtitles a somewhat cryptic error is thrown.
**To Reproduce**
1. Have no providers set in bazarr
2. Connect sonarr or radarr
3. Try to search for a subtitle
4. See the following error in logs
```
Exception on /api/movies/subtitles [PATCH]
```

```
Traceback (most recent call last):
  File "/app/bazarr/bin/bazarr/../libs/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/app/bazarr/bin/bazarr/../libs/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_ars)
  File "/app/bazarr/bin/bazarr/../libs/flask_restful/__init__.py", line 467, in wrapper
    resp = resource(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/../libs/flask/views.py", line 84, in view
    return current_app.ensure_sync(self.dispatch_request)(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/../libs/flask_restful/__init__.py", line 582, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/api/utils.py", line 30, in wrapper
    return actual_method(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/api/movies/movies_subtitles.py", line 59, in patch
    result = list(generate_subtitles(moviePath, [(language, hi, forced)], audio_language,
  File "/app/bazarr/bin/bazarr/get_subtitle/pool.py", line 75, in decorated
    updated = _update_pool(args[args_spec.index("media_type")], profile_id)
  File "/app/bazarr/bin/bazarr/get_subtitle/pool.py", line 52, in _update_pool
    return pool.update(
  File "/app/bazarr/bin/bazarr/../libs/subliminal_patch/core.py", line 556, in update
    updated = super().update(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/../libs/subliminal_patch/core.py", line 108, in update
    updated = set(providers) != self.providers or ban_list != self.ban_list
TypeError: \'NoneType\' object is not iterable
```

**Expected behavior**
A more user friendly message explaining providers need to be added is seen in logs.
**Screenshots**

**Software (please complete the following information):**
 - Bazarr: v1.0.4
 - Radarr version 4.1.0.6175
 - OS: unraid

**Comment by morpheus65535** [comment]
@vitiko98 could you take a look please?

**Comment by vitiko98** [comment]
This was fixed two months ago (a783515ad4fc5f54a27c97a2955679c6555a5839). Try the latest release.

**Comment by morpheus65535** [comment]
> This was fixed two months ago ([a783515](https://github.com/morpheus65535/bazarr/commit/a783515ad4fc5f54a27c97a2955679c6555a5839)). Try the latest release.

My bad! Thanks!
